apps/meteor/ee/server/apps/orchestrator.js

```diff
@@ -243,21 +248,6 @@ settingsRegistry.addGroup('General', function () {
 			alert: 'Apps_Logs_TTL_Alert',
 		});
 
-		this.add('Apps_Framework_enabled', true, {
-			type: 'boolean',
-			hidden: false,
-		});
-
-		this.add('Apps_Framework_Development_Mode', false, {
```

Comment:
I found a place still using the setting: https://github.com/RocketChat/Rocket.Chat/blob/c976d00f145dbb901fa1b02df603241da29c9765/apps/meteor/app/lib/server/functions/sendMessage.js#L223

Reply:
@d-gubert